src/core/udb-api.service.js

```diff
@@ -252,14 +252,15 @@ function UdbApi(
   this.formatOfferClass = function(event) {
     var offer = {};
     var type = event['@type'].toLowerCase();
-    if (type === 'event') {
-      offer = new UdbEvent();
-    }
-    else if (type === 'place') {
-      offer = new UdbPlace();
-    }
-    else {
-      offer = new UdbOrganizer();
+    switch(type) {
+      case 'event':
+        offer = new UdbEvent();
+        break;
+      case 'place':
+        offer = new UdbPlace();
+        break;
+      default:
```

Comment:
Should we explicitly use `case 'organizer'` here and do nothing on `default`?

Reply:
Yes, anything else is unexpected and should result in an error here. The rest of the UI should be able to handle it gracefully though and just not show that item (maybe log the error in the console though). Parsing unknown types  as organizers could lead to weird behavior if the back-end suddenly starts returning new or unexpected document types as a bug or change that's not picked up here yet.